src/services/api.ts

```diff
@@ -0,0 +1,36 @@
+import axios from "axios";
```

Comment:
- [ ] let's use React Query library to make API calls

Reply:
@AndriiKasian I've used RTK Query for making API calls and removed Axios from the project.